lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,40 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
```

Comment:
Agreed. We had yet to write any kind of REPL text linter until we had written enough in the wild to determine that we had settled around a particular set of conventions. We are probably at a point where we could write a formal linter.